docs/content/admin/quickstart.mdx

```diff
@@ -33,79 +33,146 @@ You can install Node from the [official website](https://nodejs.org/en/).
 
 ---
 
-## Instant Deployment to Netlify
+## Install the Admin
 
-Instead of manually following this guide to install then later deploy the Medusa Admin, you can deploy the Medusa Admin to Netlify with this button:
+### Step 1: Install the Package
 
-<a href="https://app.netlify.com/start/deploy?repository=https://github.com/medusajs/admin" className="img-url">
-  <img src="https://www.netlify.com/img/deploy/button.svg" alt="Deploy to Netlify" className="no-zoom-img" />
-</a>
+In the directory of your Medusa backend, run the following command to install admin dashboard:
 
----
+```bash npm2yarn
+npm install @medusajs/admin
+```
 
-## Install the Admin
+### Step 2: Add Admin to Medusa Configurations
+
+In `medusa-config.js`, add the admin plugin into the array of `plugins`:
+
+```js title=medusa-config.js
```

Comment:
note: If a user intends to deploy the admin separately (e.g. Vercel), they should not add anything to their `medusa-config`.